tests/Games/NexusMods.Games.StardewValley.Tests/GenericInstallerTests.cs

```diff
@@ -0,0 +1,133 @@
+using System.Runtime.CompilerServices;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Library.Models;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.NexusWebApi.Types.V2;
+using NexusMods.Extensions.BCL;
+using NexusMods.Games.StardewValley.Installers;
+using NexusMods.Games.StardewValley.Models;
+using NexusMods.Games.TestFramework;
+using NexusMods.StandardGameLocators.TestHelpers;
+using Xunit.Abstractions;
+
+namespace NexusMods.Games.StardewValley.Tests;
+
+[Trait("RequiresNetworking", "True")]
+public class GenericInstallerTests : ALibraryArchiveInstallerTests<GenericInstallerTests, StardewValley>
+{
+    private readonly GenericInstaller _installer;
+
+    public GenericInstallerTests(ITestOutputHelper outputHelper) : base(outputHelper)
+    {
+        _installer = ServiceProvider.GetRequiredService<GenericInstaller>();
+    }
+
+    protected override IServiceCollection AddServices(IServiceCollection services)
+    {
+        return base.AddServices(services)
+            .AddStardewValley()
+            .AddUniversalGameLocator<StardewValley>(new Version("1.6.14"));
+    }
+
+    [Fact]
+    public async Task Test_NotSupported()
+    {
+        var loadout = await CreateLoadout();
+
+        // 3D NPC Houses 1.0 https://www.nexusmods.com/stardewvalley/mods/763?tab=files
+        var libraryArchive = await DownloadArchiveFromNexusMods(ModId.From(763), FileId.From(2874));
+
+        using var tx = Connection.BeginTransaction();
+        var group = new LoadoutItemGroup.New(tx, out var id)
+        {
+            IsGroup = true,
+            LoadoutItem = new LoadoutItem.New(tx, id)
+            {
+                Name = "Foo",
+                LoadoutId = loadout,
+            },
+        };
+
+        var result = await _installer.ExecuteAsync(libraryArchive, group, tx, loadout, CancellationToken.None);
+        result.IsNotSupported.Should().BeTrue();
+    }
+
+    [Theory]
+    [InlineData(1915, 124659,1)] // Content Patcher 2.5.3 (https://www.nexusmods.com/stardewvalley/mods/1915?tab=files)
+    [InlineData(16893, 123812,2)] // Romanceable Rasmodius Redux Revamped 1.8.55 (https://www.nexusmods.com/stardewvalley/mods/16893?tab=files)
+    [InlineData(18144, 114038,1)] // Romanceable Rasmodia - RRRR Patch 1.1 (https://www.nexusmods.com/stardewvalley/mods/18144?tab=files)
+    [InlineData(31167, 123427,0)] // Item Bags for Stardew Valley Expanded 1.0.0 (https://www.nexusmods.com/stardewvalley/mods/31167?tab=files)
+    [InlineData(20414, 126173, 2)] // Portraits for Vendors 1.9.3 - Nyapu's Portraits (https://www.nexusmods.com/stardewvalley/mods/20414?tab=files)
```

Comment:
This test is not checking that the `Credits.xlsx` file is actually being installed correctly (not being 0 size etc).
Couldn't you just add a separate test copying over the old test code?

The reference ticket for the issue was this one:
- #2367